src/ofxEmotiBitVersion.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 //#include <string>
 #include "ofMain.h"
-const std::string ofxEmotiBitVersion = "1.4.4";
+const std::string ofxEmotiBitVersion = "1.4.4-feat-advertisingOptions.6";
```

Comment:
Is this the correct convention for version bump?

I thought it was
`x`.`x`.`x`.`branch`-`interationOfBranch`

Reason being, the delimiter for version is a `.`

Reply:
We need to write it down. I'll add it as a task. It is a trivial change for now, so I'm resolving this comment!